src/components/tx/ApprovalEditor/ApprovalValueField.tsx

```diff
@@ -77,13 +85,11 @@ export const ApprovalValueField = ({ name, tx }: { name: string; tx: ApprovalInf
                 paddingBottom: '4px',
                 paddingLeft: 1,
                 flexWrap: 'nowrap !important',
+                '&::before': {
+                  border: 'none !important',
+                },
+                border: 'none !important',
```

Comment:
I think we can also use `variant="standard"` to achieve this look and we could also remove the custom paddings then
<img width="638" alt="Screenshot 2024-04-26 at 14 20 01" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/b54193e7-9363-4142-97f8-3f55679ce0b0">